src/txn/txn_rollback_to_stable.c

```diff
@@ -1030,8 +1033,10 @@ __rollback_to_stable_btree_hs_truncate(WT_SESSION_IMPL *session, uint32_t btree_
          * we can skip it.
          */
         if (__wt_txn_visible_all(
-              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts))
+              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts)) {
```

Comment:
You can use the helper function __wt_txn_tw_stop_visible_all for this.

Reply:
52fe369